ui/src/sources/components/InfluxTableRow.tsx

```diff
@@ -36,10 +37,11 @@ class InfluxTableRow extends PureComponent<Props & WithRouterProps> {
       setActiveKapacitor,
       deleteKapacitor,
       toggleWizard,
+      testId,
     } = this.props
 
     return (
-      <tr className={this.className}>
+      <tr className={this.className} data-test={testId}>
```

Comment:
could be simply data-test="connection-row" without the need to use testId, since it is never parametrized